modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/pages/Workspaces.tsx

```diff
@@ -91,6 +94,17 @@ const WorkspacesContent = ({
 
 const Workspaces: any = () => {
 	const {data: projects, loading} = useFetchProjects();
+
+	const {data: preferences, loading: preferencesLoading} = useIncidentAlert();
+
+	const [showIncidentAlert, setShowIncidentAlert] = useState(false);
+
+	useEffect(() => {
+		if (!preferencesLoading && preferences.incidentAlertEnabled) {
+			setShowIncidentAlert(true);
+		}
+	}, [preferencesLoading, preferences.incidentAlertEnabled]);
```

Comment:
Create onClose fn to close the alert and remove it

Reply:
done